packages/cli/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@janus-idp/cli",
+  "name": "@janus-idp/backstage-cli",
   "description": "CLI for developing Backstage plugins and apps",
-  "version": "1.0.5",
+  "version": "1.0.6-next.0",
```

Comment:
This is versioned automatically by the release toolkit and released on merge

Reply:
OK. So what should I do here ? Just rollback this change ?

Reply:
I rolled back the change